addons/l10n_id_pos/models/pos_order.py

```diff
@@ -0,0 +1,8 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    l10n_id_qris_transaction_ids = fields.Many2many('l10n_id.qris.transaction')
```

Comment:
Did you make a mistake of not using this field? I don't see its use in `l10n_id_pos`.

Reply:
The field is still necessary in my opinion because of https://github.com/odoo-dev/odoo/blob/master-qris_pos-nni-dev/addons/l10n_id/models/res_bank.py#L108-L110

the `trx_record` here could refer to `account.move` and `pos.order` (or any other models in the future that involves QR payment as well).

Reply:
Can you put a comment on it so that we don't forget that it's referenced indirectly?

Reply:
Got it!